shared_model/backend/protobuf/queries/impl/proto_get_account_asset_transactions.cpp

```diff
@@ -39,5 +41,11 @@ namespace shared_model {
       return account_asset_transactions_.asset_id();
     }
 
+    std::unique_ptr<interface::TxPaginationMeta>
```

Comment:
Why do we want to return a unique_ptr? Is the meta going to be ever changed? If not, why not returning a const ref, as it is done in other methods of the class?

Reply:
The other methods return primitive objects, while this requires construction. Returning a constref would require us to keep a copy of `proto::TxPaginationMeta` inside all proto query classes. I think a unique pointer is a more clear way to provide `proto::TxPaginationMeta` when it is needed.

Reply:
Please, consider this issue again before merging

Reply:
Here we would need to create an object every time pagination meta object is invoked, whereas it is only access methods. Please change to cont ref as it is done for example for amount in transfer asset. It is okay to construct  pagination meta